vhdbuilder/packer/init-variables.sh

```diff
@@ -182,6 +182,18 @@ if [ ! -z "${WINDOWS_SKU}" ]; then
 	esac
 fi
 
+# no source image on marketplace, use MSFT/AME SIG as source image
+if [[ ${ARCHITECTURE,,} == "arm64" ]]; then
+  ARM64_SIG_SUBSCRIPTION_ID="a84c0852-ea2d-4359-88e4-11a80a4fb6b9"
```

Comment:
how will we access this, is the image public?

Reply:
I asked the ARM64 team to grant read access to aksvhdbuilderSP for the AME AMR64 SIG,  aksvhdbuildSP and aks_agentbaker_vhd_test for the MSFT ARM64 SIG. The packer build pipeline can create the ARM64 VMs, start the script install-dependencies.sh. I will make changes in install-dependencies.sh and file related in following PRs to keep each PR not too big.